asciidoc 10 fails with "missing configuration file"

In our build process, we call a2x as follows:

> `a2x -vv -L --asciidoc-opts="-f ../build/mscgen-filter.conf -f ../build/diag-filter.conf -f ../build/docinfo-releaseinfo.conf -a srcdir='/home/user/code/osmo-dev/src/osmo-gsm-manuals/tests' -a commondir='../common'" --dblatex-opts="-s ../build/custom-dblatex.sty -P draft.mode=yes -P draft.watermark=0" -a docinfo -a revnumber="DRAFT " -a revdate="unknown" test-usermanual.adoc`

with asciidoc 9, this works:
```
a2x: args: ['-vv', '-L', "--asciidoc-opts=-f ../build/mscgen-filter.conf -f ../build/diag-filter.conf -f ../build/docinfo-releaseinfo.conf -a srcdir='/usr/src/packages/BUILD/tests' -a commondir='../common'", '--dblatex-opts=-s ../build/custom-dblatex.sty -P draft.mode=yes -P draft.watermark=0', '-a', 'docinfo', '-a', 'revnumber=DRAFT ', '-a', 'revdate=unknown', 'test-usermanual.adoc']
a2x: resource files: []
a2x: resource directories: ['/etc/asciidoc/images', '/etc/asciidoc/stylesheets']
a2x: executing: "/usr/bin/asciidoc" --backend docbook -a "a2x-format=pdf" -f ../build/mscgen-filter.conf -f ../build/diag-filter.conf -f ../build/docinfo-releaseinfo.conf -a srcdir='/usr/src/packages/BUILD/tests' -a commondir='../common' --attribute "docinfo" --attribute "revnumber=DRAFT " --attribute "revdate=unknown" --verbose  --out-file "/usr/src/packages/BUILD/tests/test-usermanual.xml" "/usr/src/packages/BUILD/tests/test-usermanual.adoc"
...
```

with asciidoc 10, it fails:
```
a2x: args: ['-vv', '-L', "--asciidoc-opts=-f ../build/mscgen-filter.conf -f ../build/diag-filter.conf -f ../build/docinfo-releaseinfo.conf -a srcdir='/usr/src/packages/BUILD/tests' -a commondir='../common'", '--dblatex-opts=-s ../build/custom-dblatex.sty -P draft.mode=yes -P draft.watermark=0', '-a', 'docinfo', '-a', 'revnumber=DRAFT ', '-a', 'revdate=unknown', 'test-usermanual.adoc']
a2x: resource files: []
a2x: resource directories: ['/etc/asciidoc/stylesheets']
make[3]: *** [../build/Makefile.asciidoc.inc:70: test-usermanual.pdf] Error 1
```
It does not print an error message (at least when called through make), only exits with 1.

I've traced the code to find that it it crashes here:

https://github.com/asciidoc-py/asciidoc-py/blob/74823bed08e7b823d2b6cba8636a1fd67fed6cc9/asciidoc/asciidoc.py#L5959

When adding a print right before the exception with the message, I get the error message:

> `missing configuration file: ../build/mscgen-filter.conf -f ../build/diag-filter.conf -f ../build/docinfo-releaseinfo.conf -a srcdir='/home/user/code/osmo-dev/src/osmo-gsm-manuals/tests' -a commondir='../common'`

So it seems that with asciidoc 10, it is not parsing the configuration files correctly from the command-line.

--- a/asciidoc/a2x.py
+++ b/asciidoc/a2x.py
@@ -980,7 +980,7 @@ def cli():
     opts, args = parser.parse_args(argv)
     if len(args) != 1:
         parser.error('incorrect number of arguments')
-    opts.asciidoc_opts = [x.split(' ', 1) for x in opts.asciidoc_opts]
+    opts.asciidoc_opts = [x.split(' ') for x in opts.asciidoc_opts]
     opts.dblatex_opts = ' '.join(opts.dblatex_opts)
     opts.fop_opts = ' '.join(opts.fop_opts)
     opts.xsltproc_opts = ' '.join(opts.xsltproc_opts)

